pyvrp/tests/test_Population.py

```diff
@@ -143,15 +144,15 @@ def test_select_returns_same_parents_if_no_other_option():
     rng = XorShift128(seed=2_147_483_647)
 
     params = PopulationParams(min_pop_size=0)
-    pop = Population(data, pm, rng, broken_pairs_distance, params)
+    pop = Population([], bpd, params=params)
```

Comment:
I was thinking about whether we should make `params` a keyword only argument. Perhaps yes, but then not in this PR (since we should then probably enforce it all at once for every Params-like object).